### PR TITLE
[Central] - Pagination from new Api

### PR DESCRIPTION
**Pull request by drewjhart**
WIP

### CODE REVIEW

File: central/src/containers/RouteContainer.tsx [hunk 1]
```diff
@@ -272,29 +291,27 @@ export const RouteContainer = ({
...
-      <GamesFooter listGameTemplateNextToken={listGameTemplateNextToken} listGameTemplatePrevTokens={listGameTemplatePrevTokens} handleNextPage={handleNextPage} handlePrevPage={handlePrevPage}></GamesFooter>
+      <Nav isResolutionMobile={isResolutionMobile} isUserAuth={isUserAuth} handleModalOpen={handleModalOpen} />
+      <Games loading={loading} nextToken={nextToken} games={filteredGames} questions={questions} handleScrollDown={handleScrollDown} saveNewGame={saveNewGame} saveGame={saveGame} updateQuestion={updateQuestion} deleteQuestion={handleDeleteQuestion} deleteGame={handleDeleteGame} cloneGame={handleCloneGame} sortType={sortType} setSortType={setSortType} cloneQuestion={cloneQuestion} isUserAuth={isUserAuth}  isSearchClick={isSearchClick} handleSearchClick={handleSearchClick} setSearchInput={setSearchInput} searchInput={searchInput} isResolutionMobile={isResolutionMobile} addQToGT={addQToGT} handleQuestionBankClick={handleQuestionBankClick} />
```
Comment (maniramezan) [inline central/src/containers/RouteContainer.tsx:314]: Probably better to break it into multiple lines for readability

File: central/src/components/QuestionDashboard.tsx [hunk 2]
```diff
@@ -1,26 +1,56 @@
...
+            />
+            <Typography className={classes.loadingTitle}>
+              Loading Question List...
+            </Typography>
+            <Typography className={classes.loadingText}>
+              If there are issues with loading, try reloading this page.
+            </Typography>
+          </div>
+        </div>
+              </>
```
Comment (maniramezan) [inline central/src/components/QuestionDashboard.tsx:46]: nit: indent

File: central/src/containers/RouteContainer.tsx (hunk 1 above)
Comment (drewjhart) [reply central/src/containers/RouteContainer.tsx]: fixed

File: central/src/components/QuestionDashboard.tsx (hunk 2 above)
Comment (drewjhart) [reply central/src/components/QuestionDashboard.tsx]: fixed

**Pull request merged**